### ISSUE TITLE
Menu 

### ISSUE THREAD

**Issue by MikeWalrath** [hpe-design/design-system#517]
Acceptance criteria is based on alignment across the below 
- [x] Figma 
- [x] Theme
- [x] Grommet
- [x] Code Examples
- [x] Guidance

**Comment by taysea** [comment]
To guide any component day discussions:
Keyboard focus behavior:
- Only one element on the page should have focus indication at a time.
- If a Menu is open, it seems like the menu item in focus (the one that would be read by a screen reader) should have focus, and not the main menu button. This would require a design change
- According to this resource, it seems that our hover styling can also clear as focus styling.
- Resource: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html
Q: Does having focus outline around the main menu button and the light grey background on hovered/keyboard accessed menu item conflict with the notion that only one item should be in focus at a time? Q: What should be the focused element from a DOM perspective as I use keyboard to navigate through the menu?
	- If so, suggestion: Don’t have focus outline around menu button if an item is active via keyboard
Q: How would this principle propagate to other components such as Select, TextInput, etc.?

**Comment by ericsoderberghp** [comment]
Some guidance starters:
- limit the number of options in the Menu to 6 or less.
- No cascading menus?
- keep the labels short
- avoid labelling the menu with "menu", put something more descriptive about what it contains or the context it is in

**Comment by MikeWalrath** [comment]
@taysea @vavalos5 Looks like theres a discrepancy between the color of the carrot in the theme or on Figma. Not sure how we missed this when we reviewed but wanted to call it out.

**Comment by taysea** [comment]
The keyboard behavior of Menu needs to be implemented in Grommet. I have opened an issue for there as #734 . The Menu focus designs were updated within the last couple days and the new designs don't align with previous decisions on Menu focus behavior. So, we will need to discuss this further next week before changes in Grommet can be made.
